### ISSUE TITLE
[Feature] Cache getRecs results when navigating to other pages

### ISSUE THREAD

**Issue by SaturnusDJ** [tinderjs/tinder-desktop#20]
At this moment when you go from swipes to somewhere else and then returning to the swipe page, another swipe card is displayed. In other words, the swipe process is re-initiated.
I believe this is one of the causes that eventually locks up the swiping and resulting in the 'Out of people' messages while there actually are enough swipes left. And even if it's not, it's still not so nice.

Suggestion: code to keep swipe page running in background.

**Comment by mayeaux** [comment]
Yes I agree with you 100%, and you're actually right that this is what is causing the 'Out of people' thing to appear The results from the initial .getRecs call should be cached until they've all had their card thrown (ie been swiped left or right or superliked). The code base should lend itself to this feature as well so I'm +100 on this.
